torch/_inductor/graph.py

```diff
@@ -492,6 +492,10 @@ def __init__(
         self.aligned_inputs = OrderedSet[str]()
         self.no_fuse_buffer_names = OrderedSet[str]()
 
+        self.low_precision_codegen_ops = OrderedSet[str]()
+        # more aggressive prologue fusion
+        self.invoke_quant_ops = OrderedSet[str]()
```

Comment:
```suggestion
        self.low_precision_codegen_ops : OrderedSet[str] = OrderedSet()
        # more aggressive prologue fusion
        self.invoke_quant_ops : OrderedSet[str] = OrderedSet()
```
to avoid runtime calls to `__getitem__`.